bin/vsm.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```

Comment:
Well the setup.py if you want to install the project will skip that file, I have to do some jiggery to remove the .py just hadn't started it yet because we are still in flux.